### ISSUE TITLE
Inquiry about the traning code

### ISSUE THREAD

**Issue by LanSaber** [zhangy76/PhysPT#2]
Impressive work! Would you please upload the training codes？ I am very curious about how you implement the loss function and get these physics quantities.

**Comment by zhangy76** [comment]
Thanks for your interests. I may not have time recently to clean the code and upload it, sorry about this! But for the physics quantities, we can obtain them from the output of in models/smpl_phys.py. These physical quantities are basically all we need to compute the losses.

**Comment by zhangy76** [comment]
Close as being inactive. Feel free to open it again, thanks.
